packages/ts/react-crud/src/autoform.tsx

```diff
@@ -344,7 +352,7 @@ export function AutoForm<M extends AbstractModel>({
           </Button>
         ) : null}
         {deleteButtonVisible && isEditMode && (
-          <Button className="auto-form-delete-button" theme="tertiary error" onClick={deleteItem}>
+          <Button className="auto-form-delete-button" theme="tertiary error" onClick={deleteItem} typeof="submit">
```

Comment:
What does `typeof` do here? I can't find any reference to such an attribute existing. Also why is the delete button type of "submit"?